vector/src/main/res/values/strings.xml

```diff
@@ -597,7 +597,7 @@
     <string name="_continue">Continue</string>
 
     <!-- Room -->
-    <string name="list_members">List members</string>
+    <string name="list_members">Members</string>
```

Comment:
According to the [doc](https://github.com/vector-im/element-android/blob/main/CONTRIBUTING.md#editing-existing-strings), it's OK to edit the string. We are in case 1:

> If the meaning stays the same, it's OK to edit the original string (i.e. the English version).

What is not described in the doc, is that Weblate will detect automatically the change and mark all the translations as "need update" (or something similar). So we will see updates of those translations in the future PRs from Weblate.